parquet-thrift/src/main/java/org/apache/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -325,7 +323,7 @@ private ConvertedField visitPrimitiveType(PrimitiveTypeName type, LogicalTypeAnn
 
   @Override
   public ConvertedField visit(EnumType enumType, State state) {
-    return visitPrimitiveType(BINARY, enumType(), state);
+    return visitPrimitiveType(BINARY, LogicalTypeAnnotation.enumType(), state);
```

Comment:
Why you don't like static imports? This change (and the removing of the related import) seems to be useless.

Reply:
I find accessing these via `LogicalTypeAnnotation.enumType()` more clear than just `enumType()`. The larger change in this PR allows any thrift type to use a LogicalTypeAnnotation, so importing the class opens the door to further changes that implement these logical types.

Reply:
Yes, I know what this PR is about. My comment was only about this line (and the removing of the static import).

It is a matter of taste whether you like static imports or not and it is up to you if you use them or not. But modifying an existing line is another topic. I usually like to keep a change as small as possible and do not touch other code parts if it is not necessary.
Anyway, if you have a strong opinion that this change improves readability I am fine with that too.

Reply:
I remove this static import for consistency with the logical types in my changes. I do find qualifying the type with `LogicalTypeAnnotation.` improves readability.

If we prefer a set of discrete static imports to importing the larger class I can certainly do that instead. My motivation was more concise imports and readability.

Reply:
I accept it as a strong opinion :)